CHANGELOG.md

```diff
@@ -15,6 +15,19 @@ it according to semantic versioning. For example, if your PR adds a breaking cha
 should change the heading of the (upcoming) version to include a major version bump.
 
 -->
+# 5.8.0
```

Comment:
Since we haven't published `5.7.0` yet this could be released as part of that release. I have one more PR to complete for it and won't be done with it until early next week.